test/integration/packwerk/signatures_load_test.rb

```diff
@@ -0,0 +1,13 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "support/rails_test_helper"
+
+module Packwerk
+  class SignatureLoadTest < Minitest::Test
+    test "run all sig blocks without crashing" do
+      %x(bundle exec ruby -Ilib -rbundler/setup -rpackwerk -e "T::Utils.run_all_sig_blocks")
```

Comment:
Instead, can we use `ActiveSupport::Testing::Isolation`?

Reply:
TIL about `Testing::Isolation`, neat! I don't think it works here though because it uses forking, which already loaded the required code, unless I'm doing it wrong? Here's what I have:

```ruby
include ActiveSupport::Testing::Isolation`

test "run all sig blocks without crashing" do
  run do
    T::Utils.run_all_sig_blocks
  end
end
```

Commenting out the fix, this test still passes.

Reply:
Gotcha. Looking into this again though, `-rbundler/setup` and `bundle exec` should not be both required – only one. I'll fix that.